packages/teams-js/README.md

```diff
@@ -24,7 +24,7 @@ To install the latest [2.0 preview version](https://docs.microsoft.com/en-us/jav
 
 ### Production
 
-You can access [these files on unpkg](https://res.cdn.office.net/teams-js/2.0.0-beta.6/js/MicrosoftTeams.min.js), download them, or point your package manager to them.
+You can access [these files on unpkg](https://res.cdn.office.net/teams-js/2.0.0-beta.7/js/MicrosoftTeams.min.js), download them, or point your package manager to them.
```

Comment:
>unpkg [](http://example.com/codeflow?start=31&length=5)

Just noticed this in the readme. Technically these aren't on unpkg.

Reply:
Will fix before the next release.